config/database.yml

```diff
@@ -21,7 +21,7 @@ default: &default
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
```

Comment:
- [ ] Please make sure that the project works properly. Currently, when I tried to set up the database for testing, I got the following error which prevented me from running and testing the app.

![image](https://user-images.githubusercontent.com/62290545/232625433-dc4af16d-3784-43bc-b41f-1dfbb8c06c6a.png)

- [HINT] If you want to simulate showing the error, try dropping the database and creating it again.

Reply:
Done! Thanks a lot! @ShadyShawkat